locales/fr/instructions.json

```
@@ -0,0 +1,6 @@
+{
+	"title": "Instructions",
+	"description": "Vous allez recevoir une série de décalrations. Pour cacune d'elles, cliquez sur l'option qui correspond le mieux à votre opinion",
+	"confirm": "Je l'ai eu!",
+	"cancel": "Attends, laisse tomber!"
```

The rest of the French translations are using the formal tone.

```suggestion
	"cancel": "Attendez, laissez tomber!"
```